flow/packet-headers/icmp.yaml

```diff
@@ -0,0 +1,47 @@
+components:
+  schemas:
+    Flow.Icmp:
+      description: ICMP packet header
+      type: object
+      properties:
+        type:
+          x-field-pattern:
+            x-constants:
+              echo_reply: 0
+              echo_request: 8
+              destination_unreachable: 3
+              source_quench: 4
+              redirect_message: 5
+              time_exceeded: 11
+              parameter_problem: 12
+              timestamp: 13
+              timestamp_reply: 14
+              information_request: 15
+              information_reply: 16
+            description: >-
+              ICMP Type
+            format: integer
+            length: 8
+            default: 4
```

Comment:
is this the correct default?

Reply:
Sorry this should be 3 i.e. destination unreachable. I modified it.

Reply:
I think it should default to `echo_request`

Reply:
Modified it.